bgapi/module.py

```diff
@@ -207,7 +208,18 @@ def get_services(self):
     def get_characteristics(self):
         return [j for i, j in sorted(self.characteristics.items(), key=operator.itemgetter(0))]
 
+    def get_gatt_database(self):
+        return self.gatt_database
+
     def update_uuid(self, handle, uuid):
+        # TODO: I am sure there is a better way to do this
+        uuid_hex_val = 0
+        for i in uuid[::-1]:
+            uuid_hex_val <<= 8
+            uuid_hex_val |= ord(i)
+        handle_dict = {"Handle" : handle, "UUID" : uuid_hex_val}
+        if handle_dict not in self.gatt_database:
+            self.gatt_database.append({"Handle" : handle, "UUID" : uuid_hex_val})
```

Comment:
Yes, self.handle_uuid[] and self.uuid_handle[] is just organized version of self.gatt_database[], different representation. I like printing items in self.gatt_database[] to see the GATT database as a whole, if you do not like it, I can remove it.